### DISCUSSION TITLE
Simplisafe Version 2

### DISCUSSION THREAD

**Discussion by laurentfignon** [homebridge-simplisafe3/homebridge-simplisafe3#175] [Q&A] [answered]
Hi -- new user here.  I have a Simplisafe Version 2 -- When I install & provide login, it gives me (surprise!) this error: "Base station version was not 3!"  Does anyone know why Version 3 is necessary or can point me in the general direction of a solution?  Thanks!

**Comment by shamoon** [comment, marked as answer]
Well, explicitly this plugin does not support version 2, I believe the last one that did was: https://github.com/chowielin/homebridge-simplisafe-security-system but it is not actively maintained so not sure about compatibility with newest versions of Homebridge.

Duplicate of https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/issues/46

**Reply by laurentfignon** [reply]
Thank you!  I hope this might be a simple question that might help me find a solution, but do you know *why* the new version doesn't support version 2?  Is the API different for the 2 versions or something? (if so, can you point me to any documentation for the V2 API?) Or do you know where that error is coming from?  Sorry for being a bit clueless...  Many thanks again.

**Reply by laurentfignon** [reply]
I should note that I did try the chowielin version -- it didn't get me past the 2-factor login...

**Reply by shamoon** [reply]
Yea I think that message is returned from the SS API if you try to talk to the this version of the API with an old base station. Not sure if theres documentation out there for v2, sorry.

**Reply by nazgul** [reply]
Ironic that I *finally* have time to setup HomeBridge to work with Simplisafe, and in the meantime the world has moved on and it no longer supports my system. Not really ready to spring $500 for an a nice UI. :/
